src/sysrepo.proto

```diff
@@ -244,12 +244,13 @@ message Schema {
   required string module_name = 1;           /**< Name of the module. */
   required string ns = 2;                    /**< Namespace of the module. */
   required string prefix = 3;                /**< Prefix of he module. */
-  required bool implemented = 4;             /**< TRUE for explicitly installed modules. */
+  required bool installed = 4;               /**< TRUE for explicitly installed modules. */
+  required bool implemented = 5;             /**< TRUE for implemented modules (do not have to be installed). */
```

Comment:
Please add mroe information when the not-installed module is actually implemented. Here, or/and in module_dependencies.h.